eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -632,10 +632,15 @@ case class LocalParams(nodeId: PublicKey,
                        htlcMinimum: MilliSatoshi,
                        toSelfDelay: CltvExpiryDelta,
                        maxAcceptedHtlcs: Int,
-                       isInitiator: Boolean,
+                       isChannelOpener: Boolean,
```

Comment:
I understand the rationale given in https://github.com/ACINQ/lightning-kmp/pull/649#discussion_r1626132818, but I really think we should rename it back to `isInitiator` when the dust settles.

In the meantime, the tradeoff is that we are adding a new concept of "opener", that doesn't exist in the spec, while in fact we should just add the concept of `paysCommitFee`.

Reply:
> the tradeoff is that we are adding a new concept of "opener", that doesn't exist in the spec

On the contrary, this concept does exist in the spec and is used: if both sides send `stfu` concurrently, the one who gets to be the initiator for the quiescence session is the channel opener. We do use that in our implementation.

Reply:
Fair enough!